x-pack/plugins/search_playground/public/components/app.tsx

```diff
@@ -23,7 +26,7 @@ export const App: React.FC = () => {
       paddingSize="none"
       className="eui-fullHeight"
     >
-      <Chat />
+      {showStartPage ? <StartNewChat onStartClick={() => setShowStartPage(false)} /> : <Chat />}
```

Comment:
Rather than depending on a state variable, should we use session storage to keep track if we are coming to `playground` with default index or is there any indices already selected and we can directly move to `<chat/>` component? 
cc: @joemcelroy @TattdCodeMonkey

Reply:
I didn't change logic behind it.
I just moved to a parent component as it seemed a more appropriate place.

Reply:
Yes, definitely not looking to change in this PR. It could be a future improvement we can make.